creator/studies/migrations/0030_fix_populate_domain.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.2.24 on 2021-11-02 18:23
+
+"""
+This migration creates the domain attribute for the Study model and also
+populates it using the spreadsheet found in this migrations folder.
+"""
+
+from django.db import migrations, models
+
+import pandas as pd
+import os
+
+WRONG_DOMAIN = "CANCERBIRTHDEFECT"
+CORRECT_DOMAIN = "CANCERBIRTHANDDEFECT"
```

Comment:
should this be
`CANCERANDBIRTHDEFECT`?

Reply:
Ughhhhh!!! Yes lol. Don't be sorry..this is easy to mess up